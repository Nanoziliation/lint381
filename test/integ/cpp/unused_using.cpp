using namespace std;

using std::cout;

using std::cin;

cin >> foo;
