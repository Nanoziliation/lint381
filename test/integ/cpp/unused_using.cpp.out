unused_using.cpp:3:12: error: Unused symbol 'cout'
using std::cout;
           ^^^^
